spec/betagouvbot/webhook_spec.rb

```diff
@@ -3,10 +3,11 @@
 
 RSpec.describe BetaGouvBot::Webhook do
   describe '/compte' do
-    let(:callback)  { 'https://bob.coop' }
-    let(:user_name) { 'bob' }
-    let(:text)      { "#{user_name} #{user_name}@email.coop password" }
-    let(:params)    { { response_url: callback, user_name: user_name, text: text } }
+    let(:callback)     { 'https://bob.coop' }
+    let(:user_name)    { 'bob' }
+    let(:text)         { "#{user_name} #{user_name}@email.coop password" }
+    let(:empty_params) { { response_url: callback, user_name: user_name } }
+    let(:valid_params) { empty_params.merge(text: text) }
```

Comment:
> I think we need a test for the empty_text case

Already done. ```text``` is always casted to string.

> This can be a shorthand for text: 'help', and output a reminder of expected parameters.

You want to use a POST endpoint to get help about the way of using it ? 😨